code/modules/antagonists/wizard/equipment/soulstone.dm

```diff
@@ -182,15 +211,15 @@
 /obj/structure/constructshell/attackby(obj/item/O, mob/user, params)
 	if(istype(O, /obj/item/soulstone))
 		var/obj/item/soulstone/SS = O
-		if(!IS_CULTIST(user) && !IS_WIZARD(user) && !SS.theme == THEME_HOLY)
+		if(!IS_CULTIST(user) && !IS_WIZARD(user) && !theme == THEME_HOLY)
 			to_chat(user, span_danger("An overwhelming feeling of dread comes over you as you attempt to place [SS] into the shell. It would be wise to be rid of this quickly."))
 			user.Dizzy(30)
 			return
-		if(SS.theme == THEME_HOLY && IS_CULTIST(user))
-			SS.hot_potato(user)
+		if(theme == THEME_HOLY && IS_CULTIST(user))
+			hot_potato(user)
 			return
-		SS.transfer_soul("CONSTRUCT",src,user)
-		SS.was_used()
+		transfer_soul("CONSTRUCT",src,user)
```

Comment:
gonna guess you just control f'd everything in the file and didn't actually test anything? lol

Reply:
classy, right? i usually just eyeball it to see if I need to specifically replace or if i can just do the whole file, unfortunately in this case I was definitely wrong